.gitignore

```diff
@@ -36,3 +36,5 @@ amplifytools.xcconfig
 .secret-*
 **.sample
 #amplify-do-not-edit-end
+
+.env.production
```

Comment:
Se ho ben capito il nome .env.production sarà tale anche per gli ambienti non di produzione. In tal caso preferirei utilizzare semplicemente .env

Reply:
@marco-vit-pagopa Il comando `yarn build` cerca di default `.env.production`.